app/views/admin/username_blocks/_username_block.html.haml

```diff
@@ -0,0 +1,16 @@
+.batch-table__row
+  %label.batch-table__row__select.batch-table__row__select--aligned.batch-checkbox
+    = f.check_box :username_block_ids, { multiple: true, include_hidden: false }, username_block.id
+  .batch-table__row__content.pending-account
+    .pending-account__header
+      %samp= link_to username_block.username, edit_admin_username_block_path(username_block)
+
```

Comment:
The checkbox doesn't currently have an accessible label, could `.pending-account-header` be made a label with a `for` attribute referencing the checkbox id?